### PR TITLE
Input image

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added image component

### CODE REVIEW

File: component/image/image.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.image-container {
+  height: 20rem;
+  width: 20rem;
+}
+
+.rounded-img {
+  border-radius: 50%;
+  aspect-ratio: 1/1;
+  object-fit: cover;
```
Comment (sri-rishi) [inline component/image/image.css:24]: here you don't have to use object-fit cover, You can use responsive-img class with round-img class

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Comment by amantulsyan35** [comment]
apart from the changes @sri-rishi suggested, I can't find any other changes, seems good to merge

File: component/image/image.css (hunk 1 above)
Comment (Sahil25061999) [reply component/image/image.css:24]: I have purposefully added it so user doesn't need to use different class😅

**Review by shangkaul — APPROVED** [review]
Looks good to merge!

**Pull request merged**